### ISSUE TITLE
[FormBundle] Configure extra constraints for FileUploadPagePart

### ISSUE THREAD

**Issue by dannyvw** [Kunstmaan/KunstmaanBundlesCMS#2690]
The FileUploadPagePart has only a required constraint, but it will be useful if you can configure the max file size and extensions.

**Comment by acrobat** [comment]
Hi @dannyvw,

Since v5.1 we have a command to copy the form pageparts in your project (`kuma:generate:form-pageparts`). Does that help you to set custom constraints on those fields?

**Comment by dannyvw** [comment]
@acrobat Yes that is possible but i think it would be nice if it is standard available.
